lib/php/rendition-factory/src/DocumentationDumper.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Alchemy\RenditionFactory;
+
+use Alchemy\RenditionFactory\Transformer\Documentation;
+use Alchemy\RenditionFactory\Transformer\TransformerModuleInterface;
+use Alchemy\RenditionFactory\Transformer\Video\Format\FormatInterface;
+use Symfony\Component\Config\Definition\Dumper\YamlReferenceDumper;
+use Symfony\Component\DependencyInjection\Attribute\TaggedLocator;
+use Symfony\Component\DependencyInjection\ServiceLocator;
+
+final readonly class DocumentationDumper
+{
+    public function __construct(
+        #[TaggedLocator(TransformerModuleInterface::TAG, defaultIndexMethod: 'getName')]
+        private ServiceLocator $transformers,
+        #[TaggedLocator(FormatInterface::TAG, defaultIndexMethod: 'getFormat')]
+        private ServiceLocator $formats,
+    ) {
+    }
+
+    public static function getName(): string
+    {
+        return 'rendition-factory';
+    }
```

Comment:
At the beginning I tought you would have different kind of Documentation formats (i.e. Markdown, HTML) but why having a `rendition-factory` one? This PHP lib should only provide its documentation. Maybe you can remove the tag of the service and name it `RenditionBuildConfigDocumentation` ?

Reply:
The idea was to have many "doc" services (with a common interface + tag) inside the app, bundles, modules ... and to automatically collect/render those docs in databox. But since the TagLocator does not locates services from bundles, and since there is now only 1 "documentation service": **tag removed**.
Markdown is preferred because it can easily transform to html, pdf...

Reply:
Yes I agree, let's use markdown only!